src/pages/workspace/withFullPolicy.js

```diff
@@ -98,13 +99,14 @@ export default function (WrappedComponent) {
         previousRouteName = currentRoute.name;
         previousRoutePolicyID = policyID;
 
-        const rest = _.omit(props, ['forwardedRef', 'policy']);
+        const rest = _.omit(props, ['forwardedRef', 'policy', 'policyMemberList']);
```

Comment:
Why are `policy` and `policyMemberList` removed from props, only to be added directly as props below? I think that `forwardedRef` is the only one that needs removed from `props` because it needs to be renamed to `ref`

Reply:
Honestly not sure about this one I just followed the existing pattern. I'll see if it makes a difference